packages/qdrant-loader-core/src/qdrant_loader_core/llm/factory.py

```
@@ -95,6 +95,14 @@ def create_provider(settings: LLMSettings) -> LLMProvider:
         except Exception:
             return _NoopProvider()
 
+    if provider_name == "bedrock":
+        from .providers.bedrock import BedrockProvider
+
+        try:
+            return BedrockProvider(settings)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)